xc/xenops_server_xen.ml

```diff
@@ -2354,8 +2354,9 @@ module Actions = struct
 		device_watches := IntMap.add domid [] !device_watches
 
 	let domain_disappeared xc xs domid =
+		let domain = Printf.sprintf "domain/%d" domid in
```

Comment:
We should call this `token` rather than `domain` - it'll make the code very confusing otherwise.